### PR TITLE
feat/FOROME-(1227): add loaders

### PR DESCRIPTION
**Pull request by QSdmitrioul**
[Task](https://quantori.atlassian.net/browse/FOROME-1227)

### CODE REVIEW

File: src/pages/ws/ui/control-panel/zones/components/zone-popover.tsx [hunk 1]
```diff
@@ -66,14 +69,18 @@ export const ZonePopover = ({
...
 
-      <ZoneList
-        zoneItems={zoneList.filter(item =>
-          item.toLocaleLowerCase().includes(searchValue.toLocaleLowerCase()),
-        )}
-        selectedItems={selectedZoneItems}
-        onAdd={onAdd}
-        onRemove={onRemove}
-      />
+      {isFetching && <Loader size="m" />}
```
Comment (AndreyUstyumenko) [inline src/pages/ws/ui/control-panel/zones/components/zone-popover.tsx]: isFetching ? Loader : <ZoneList...

**Comment by AndreyUstyumenko** [comment]
Loader's too large, try a smaller size
Screen Shot 2022-06-27 at 14 11 48

**Comment by AndreyUstyumenko** [comment]
Attributes list gets freeze like this for a while. Maybe if the list didn't respond yet, we'll show a regular loader in the panel? And when it's received, the progress bar starts running.
Screen Shot 2022-06-27 at 14 14 50

File: src/pages/ws/ui/control-panel/zones/components/zone-popover.tsx (hunk 1 above)
Comment (QSdmitrioul) [reply src/pages/ws/ui/control-panel/zones/components/zone-popover.tsx]: Fixed!

File: src/components/units-list/units-list.tsx [hunk 2]
```diff
@@ -15,6 +16,7 @@ export interface IUnitsListProps {
   className?: string
   isDark?: boolean
   withCharts?: boolean
+  downloadedData: number
```
Comment (Lapk1n) [inline src/components/units-list/units-list.tsx]: Without looking there it's not obvious that downloadedData is just a primitive, not a structure.

File: src/pages/filter/dtree/components/dtree-units-list.tsx [hunk 3]
```diff
@@ -111,6 +111,7 @@ export const DtreeUnitsList = observer(
         functionalUnits={functionalUnits}
         onSelect={handleUnitSelect}
         listContainerId={listContainerId}
+        downloadedData={dtreeStore.dataReady} //TODO
```
Comment (Lapk1n) [inline src/pages/filter/dtree/components/dtree-units-list.tsx]: comment left

File: src/components/units-list/units-list.tsx (hunk 2 above)
Comment (QSdmitrioul) [reply src/components/units-list/units-list.tsx]: Changed

File: src/pages/filter/dtree/components/dtree-units-list.tsx (hunk 3 above)
Comment (QSdmitrioul) [reply src/pages/filter/dtree/components/dtree-units-list.tsx]: Upper

**Comment by QSdmitrioul** [comment]
> Attributes list gets freeze like this for a while. Maybe if the list didn't respond yet, we'll show a regular loader in the panel? And when it's received, the progress bar starts running. Screen Shot 2022-06-27 at 14 14 50

Added new loader for this behavior

**Review by EvgeniySobolev — APPROVED** [review]

File: src/store/ws/zone.store.ts [hunk 4]
```diff
@@ -14,6 +14,11 @@ class ZoneStore {
   samples: string[] = []
   zone: any[] = []
 
+  private _fetchingGenes: boolean = false
```
Comment (AlMaQntr) [inline src/store/ws/zone.store.ts]: Please, wait for zone store refactoring, it will provide async stores for zones

Comment (QSdmitrioul) [reply src/store/ws/zone.store.ts]: Done

**Review by Lapk1n — APPROVED** [review]

**Pull request merged**